tests/e2e/escape-to-host/np-status-assert.yaml

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Authors of Nimbus
+
+apiVersion: intent.security.nimbus.com/v1
+kind: NimbusPolicy
+metadata:
+  name: escape-to-host-binding
+  ownerReferences:
+    - apiVersion: intent.security.nimbus.com/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: SecurityIntentBinding
+      name: escape-to-host-binding
+status:
+  numberOfAdapterPolicies: 4
```

Comment:
There is only one Kyverno Policy then how is the `numberOfAdapterPolicies` 4?

Reply:
because the nimbus-kubearmor adapter is also running in background which will create 3 policies for escape to host

Reply:
Don't you think we should also add KSPs for `escape-to-host` intent, so that it becomes an E2E test for `escape-to-host` intent?

Reply:
In e2e tests we are already running all the adapters and then running tests, so yes in this test we are evaluating kyverno-adapter basically, and we are checking the functionality of nimbus-kyverno, irrespective of other adapters, as other adapters are already being tested in another test, so I guess this scenario is valid